Dockerfile.cypress

```diff
@@ -1,4 +1,4 @@
-FROM cypress/base:14.17.0
+FROM cypress/base:16.13.0
```

Comment:
It seems like Cypress' latest Node 16 image is 16.5 unfortunately, so perhaps we target that until it's updated?
https://github.com/cypress-io/cypress-docker-images/tree/master/base

Reply:
They probably do, but the last update was over a month ago

Reply:
Updated with latest available, `16.5.0`

Reply:
`16.13.0` is now available, reverting